depscan/lib/csaf.py

```diff
@@ -1083,14 +1082,14 @@
 }
 
 TOML_TEMPLATE = (
-    "https://raw.githubusercontent.com/owasp-dep-scan/dep-scan/master/contrib"
-    "/csaf.toml"
+    "https://raw.githubusercontent.com/owasp-dep-scan/dep-scan/master"
+    "/contrib/csaf.toml"
 )
 
 ref_map = {
     r"cve-[0-9]{4,}-[0-9]{4,}$": "CVE Record",
     r"(?<=bugzilla.)\S+(?=.\w{3}/show_bug.cgi\?id=)": "Bugzilla",
-    r"https://github.com/([\w\d\-.]+/[\w\d\-.]+/security/)?advisories":
+    r"https://github.com/([\w\d\-.]+/[\w\d\-.]+/security/)?advisories": 
```

Comment:
Does GitHub advisories have the word `security` in the URL?

Reply:
@prabhu Not always - the section comprising user/repo/security is marked optional. This allows me to identify both advisories with urls like https://github.com/advisories/advisory_id AND https://github.com/user/repo/security/advisories/advisory_id. I mostly saw the latter in our related_urls results.

I would have just searched for a GHA id at the end of a url, but I sort the pattern list by length so that the most specific patterns are prioritized, plus I'm not sure if there may be other times a GHA id ends up at the end for stuff like blog posts.